tests/integration/characters/test_dkg_and_testnet_bypass.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from nucypher.characters.chaotic import (
+    NiceGuyEddie,
+    ThisBobAlwaysDecrypts,
+    ThisBobAlwaysFails,
+)
+from nucypher.characters.lawful import Ursula
+from nucypher.policy.conditions.lingo import ConditionLingo
+from tests.constants import (
+    MOCK_ETH_PROVIDER_URI,
+    MOCK_REGISTRY_FILEPATH,
+    TESTERCHAIN_CHAIN_ID,
+)
+
+
+def _attempt_decryption(BobClass, plaintext):
+    trinket = 80  # Doens't matter.
+
+    enrico = NiceGuyEddie(encrypting_key=trinket)
+    bob = BobClass(
+        registry=MOCK_REGISTRY_FILEPATH,
+        domain="lynx",
+        eth_provider_uri=MOCK_ETH_PROVIDER_URI,
+    )
+
+    ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA = 55
+
+    definitely_false_condition = {
+        "version": ConditionLingo.VERSION,
+        "condition": {
+            "chain": TESTERCHAIN_CHAIN_ID,
+            "method": "blocktime",
+            "returnValueTest": {"comparator": "<", "value": 0},
+        },
+    }
+
+    ciphertext, tdr = enrico.encrypt_for_dkg_and_produce_decryption_request(
+        plaintext=plaintext,
+        conditions=definitely_false_condition,
+        ritual_id=ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA,
+    )
+
+    decrypted_cleartext_from_ciphertext_list = bob.threshold_decrypt(
+        ciphertext=ciphertext,
+        ritual_id=ANYTHING_CAN_BE_PASSED_AS_RITUAL_DATA,
+        conditions=definitely_false_condition,
+    )
+
```

Comment:
Do you think it needs to be tested specifically offline?  The logic shows that it works internally.

But I mean, sure, not a terrible idea I guess.

Reply:
That function has separate logic that hackers may use, and therefore should be CI tested. it's unclear to me to the function gets tested otherwise - although perhaps I'm missing something.

Reply:
Oh you're right.  I was thinking of `decrypt_using_existing_decryption_request`.

Reply:
See: https://github.com/nucypher/nucypher/issues/3172